src/lib/common/statistics.cpp

```diff
@@ -211,5 +214,8 @@ void statisticsUpdate(RequestType request, Format inFormat)
   case EntitiesResponse:                                 ++noOfEntitiesResponses; break;
   case EntryPointsRequest:                               ++noOfEntryPointsRequests; break;
   case EntryPointsResponse:                              ++noOfEntryPointsResponses; break;
+  case EntityRequest:                                    ++noOfEntityRequests; break;
+  case EntityResponse:                                   ++noOfEntityResponses; break;
+
```

Comment:
Regarding statistics improvement, the following issue is opened: https://github.com/telefonicaid/fiware-orion/issues/27. However, we shouldn't address it until PO defines the exact requirements for that operation.